lib/completions/llm.rb

```diff
@@ -17,27 +17,55 @@ module Completions
     class Llm
       UNKNOWN_MODEL = Class.new(StandardError)
 
-      def self.with_prepared_responses(responses)
-        @canned_response = DiscourseAi::Completions::Endpoints::CannedResponse.new(responses)
+      class << self
+        def models_by_provider
+          # ChatGPT models are listed under open_ai but they are actually available through OpenAI and Azure.
+          # However, since they use the same URL/key settings, there's no reason to duplicate them.
+          {
+            aws_bedrock: %w[claude-instant-1 claude-2],
+            anthropic: %w[claude-instant-1 claude-2],
+            vllm: %w[mistralai/Mixtral-8x7B-Instruct-v0.1 mistralai/Mistral-7B-Instruct-v0.2],
```

Comment:
Technically we can support 

- vLLM + Llama2
- vLLM + StableBeluga2
- vLLM + Upstage Llama2